common/persistence/cassandra/history_store.go

```diff
@@ -44,11 +44,13 @@ const (
 		`tree_id, branch_id, node_id, prev_txn_id, txn_id, data, data_encoding) ` +
 		`VALUES (?, ?, ?, ?, ?, ?, ?) `
 
+	v2templateReadHistoryNodeReverseSuffix = ` ORDER BY branch_id DESC, node_id DESC`
```

Comment:
1. ` ORDER BY branch_id DESC, node_id DESC` -> `ORDER BY branch_id DESC, node_id DESC `
2. `ORDER BY branch_id DESC, node_id DESC, txn_id ASC `

Reply:
limitations of backend require to specify order by branch_id here.